### DISCUSSION TITLE
Possibility for Google Coral TPU Acceleration?

### DISCUSSION THREAD

**Discussion by tomtom215** [mcguirepr89/BirdNET-Pi#656] [Ideas]
I know this project already runs very well with all of the optimisations for audio recognition - I was curious if it would be possible to integrate a Google Coral run the model more efficiently?

https://tutorials-raspberrypi.com/using-tensorflow-lite-with-google-coral-tpu-on-raspberry-pi-4/

**Comment by epiech** [comment]
This would be nice to see.

**Reply by tphakala** [reply]
There is no point in TPU acceleration for BirdNET model, my Golang based implementation processes 3 second chunks in 500ms on a RPI 3b+, this is far faster than is required for realtime processing.
